.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-label: 🏚️ abandoned
+          stale-issue-message: >
+            This issue has been automatically marked as `🏚️ abandoned`
+            because it has not had recent activity. It will be closed if no
+            further activity occurs.
+          close-issue-message: >
+            Closing stale issue. If this issue is still relevant, please reopen it.
+          days-before-issue-stale: 30
```

Comment:
I changed my mind about this, should perhaps set it to 60 to also support holiday periods where it takes longer to have a look at an issue.